skimage/future/detect_obj/cascade.pyx

```diff
@@ -0,0 +1,836 @@
+# cython: cdivision=True
+# cython: boundscheck=False
+# cython: nonecheck=False
+# cython: wraparound=False
+# distutils: language = c++
+
+import numpy as np
+cimport numpy as cnp
+cimport openmp
+from libc.stdlib cimport malloc, free
+from libcpp.vector cimport vector
+from skimage._shared.transform cimport integrate
+
+# Use our own implementation of the function instead of libc.math.
+# Otherwise, the compilation breaks on some compilers due to Cython
```

Comment:
"Otherwise, the compilation breaks on Windows due to a CPython bug." I'm fairly sure this isn't Cython's fault.